Microsoft.DotNet.Interactive.PowerShell/Commands/GetHtmlContentCommand.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using static Microsoft.DotNet.Interactive.Kernel;
+
+namespace Microsoft.DotNet.Interactive.PowerShell.Commands
+{
+    using System.Management.Automation;
+
+    /// <summary>
+    /// Takes the the string input and turns it into an IHtmlContent that can be passed in to
+    /// Show-JupyterContent to render Html in a Jupyter cell's output.
```

Comment:
Here and elsewhere, we should align the comment naming to the method names and remove "Jupyter" since `dotnet-interactive` is available in more notebooks than Jupyter (e.g. Azure Synapse).

Reply:
removed Jupyter from as many places as possible.